exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/TraceProtoUtils.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.exporters.jaeger;
+
+import com.google.protobuf.ByteString;
+import io.opentelemetry.trace.SpanId;
+import io.opentelemetry.trace.TraceId;
+
+/** Utilities to convert Span SDK to proto representation of the Span. */
+public final class TraceProtoUtils {
+
+  private TraceProtoUtils() {}
+
+  /**
+   * Converts a TraceId into a protobuf ByteString.
+   *
+   * @param traceId the traceId to convert.
+   * @return a ByteString representation.
+   */
+  static ByteString toProtoTraceId(TraceId traceId) {
```

Comment:
Consider to move this to Adapter. Also no need to have the class public.

Reply:
will do

Reply:
ended up moving the class over to the `otoproto` package